src/plugins/ui_actions_enhanced/common/types.ts

```diff
@@ -11,6 +11,7 @@ import type { SerializableRecord } from '@kbn/utility-types';
 
 export type BaseActionConfig = SerializableRecord;
 
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
```

Comment:
I removed .eslintrc.json in that dir because the linting in IDE didn't work with it. I just put the overrides inline where needed